lib/fluent/plugin_helper.rb

```diff
@@ -0,0 +1,36 @@
+#
+# Fluentd
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+require 'fluent/plugin_helper/event_emitter'
+require 'fluent/plugin_helper/thread'
+require 'fluent/plugin_helper/event_loop'
+require 'fluent/plugin_helper/timer'
+require 'fluent/plugin_helper/child_process'
+
+module Fluent
+  module PluginHelper
+    module Mixin
```

Comment:
Do you mean to add `self.helpers` method to `Fluent::Plugin::Base`?
I think here is better, because it keeps `Base` class a bit simpler than adding all to `Base`.
We should require all helpers from anywhere either way, and require the file which has many `require` for all helpers from `Fluent::Plugin::Base`.